README.md

```diff
@@ -19,6 +17,62 @@ features:
 For more details, please see the
 [documentation](https://kubernetes-csi.github.io/docs/Setup.html#csidriver-custom-resource-alpha).
 
+## Compatibility
+
+| Latest stable release                                                                                       | Branch                                                                                  | Compatible with CSI Version                                                                | Container Image                                 | Min K8s Version | Max K8s Version |
+| ----------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ | ----------------------------------------------- | --------------- | --------------- |
+| [cluster-driver-registrar v1.0.1](https://github.com/kubernetes-csi/cluster-driver-registrar/releases/tag/v1.0.1) | [release-1.0](https://github.com/kubernetes-csi/cluster-driver-registrar/tree/release-1.0) | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1 | 1.13            | -               |
+
+## Usage
+
+* CSI driver UNIX domain socket:
+  * Used by cluster-driver-registrar to interact with the CSI driver.
+  * Created by the CSI driver.
+  * This is the socket referenced by the `--csi-address` argument.
+* Skip attach:
+  * The cluster-driver-registrar can indicate whether the associated driver
+    requires attachment of its volumes, that is whether or not the driver
+    implements ControllerPublishVolume().
+  * This is automatically detected and requires no additional configuration.
+* Pod info on mount:
+  * The cluster-driver-registrar can indicate that the associated driver wants
+    kubelet to send additional information about the driver Pod as
```

Comment:
should we document all the supported metadata here, or point to the csi docs?

Reply:
Are there any other docs that outline these values? If so we should point to them, otherwise made this the official place where they're documented.

Reply:
I wasn't able to find specific documentation anywhere else.  I discussed with @saad-ali and he thinks that the detailed values should be in the csi-docs, and here we can point to it.

But the doc isn't ready yet :(  So maybe we just leave a todo here